packages/apps/graph/src/graph/objects/Block.ts

```diff
@@ -10,12 +10,11 @@ export default builder.prismaNode('Block', {
   fields: (t) => ({
     // database fields
     hash: t.exposeID('hash'),
-    chainid: t.expose('chainid', { type: 'BigInt' }),
```

Comment:
The GraphQL fields should reflect the changes as well, so this should also be chainId in the JS. (same for all other changed properties as well)

Reply:
I believe that has already been reflected (it gives comp error otherwise). This comment is on a removed piece of code.